specs/waku/waku-1.md

```diff
@@ -314,11 +314,11 @@ Each node SHOULD respect rate limits advertised by its peers. The number of pack
 
 ##### Light Node Field
 
-Light Node represents a boolean option. When true, the peer SHOULD NOT forward Envelopes through the Messages packet (`0x01`).
+When the `light-node` field is set to true, the host is communicating to its peers that it will not forward Envelopes through the Messages packet (`0x01`).
 
 ##### Confirmations Enabled Field
 
-Confirmations enabled represents a boolean option. When true, the peer SHOULD send [message confirmations](#batch-acknowledge-and-message-response).
+When the `confirmations-enabled` field is set true, the host is communicating to its peers that it will send [message confirmations](#batch-acknowledge-and-message-response).
```

Comment:
I don't really like this change. It removed rfc2119 language.

Reply:
It was me that added that language in the first place. I'll think of a way to rephrase this in a way that includes rfc2119